packages/@ourworldindata/grapher/src/controls/ContentSwitchers.tsx

```diff
@@ -108,24 +100,114 @@ export class ContentSwitchers extends React.Component<{
         }))
     }
 
-    render(): React.ReactElement {
-        const { manager, tabLabels } = this
+    @computed private get activeTabIndex(): number {
+        const { activeTab } = this.manager
+        if (!activeTab) return 0
+        return this.availableTabs.indexOf(activeTab) ?? 0
```

Comment:
The `??` here is useless, since `.indexOf` will return -1 if an item is not found - not undefined.

Reply:
nice catch!